Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelementryConnectionImpl.java

```diff
@@ -0,0 +1,29 @@
+package tddmicroexercises.telemetrysystem;
+
+import java.util.Random;
+
+public class TelementryConnectionImpl implements TelementryConnection {
+
+    private boolean onlineStatus;
+
+    TelementryConnectionEventSimulatorImpl telementryConnectionEventSimulator  = new TelementryConnectionEventSimulatorImpl(42);
+
+    private final Random connectionEventsSimulator = telementryConnectionEventSimulator.getConnectionEventsSimulator();
```

Comment:
make telementryConnectionEventSimulator private and final
And both these fields get objects from Constructor and initilise